packages/debug/src/browser/debug-session.tsx

```diff
@@ -1,5 +1,5 @@
 // *****************************************************************************
-// Copyright (C) 2018 Red Hat, Inc. and others.
+// Copyright (C) 2018-2023 Red Hat, Inc. and others.
```

Comment:
minor: I don't think we generally update the license headers when a file is modified.

Reply:
Ah, force of habit, thanks!